src/app/helptext/storage/volumes/datasets/dataset-acl.ts

```diff
@@ -7,50 +7,63 @@ dataset_acl_path_placeholder: T('Path'),
 dataset_acl_tag_options: [
                            {label:'User', value: 'USER'},
                            {label:'Group', value: 'GROUP'},
                            {label:'owner@', value: 'owner@'},
                            {label:'group@', value: 'group@'},
                            {label:'everyone@', value: 'everyone@'}
                          ],
+
 dataset_acl_type_placeholder: T('ACL Type'),
-dataset_acl_type_tooltip: T(''),
+dataset_acl_type_tooltip: T('How the <b>Permissions</b> are applied to\
+ the chosen <b>Tag</b>. Choose <i>Allow</i> to grant the specified\
+ permissions and <i>Deny</i> to restrict the specified permissions.'),
 dataset_acl_type_options: [
                            {label: 'Allow', value: 'ALLOW'},
                            {label: 'Deny', value: 'DENY'}
                          ],
 
 dataset_acl_perms_type_placeholder: T('Permissions Type'),
-dataset_acl_perms_type_tooltip: T(''),
+dataset_acl_perms_type_tooltip: T('Choose the type of permissions.\
+ <i>Basic</i> shows general permissions. <i>Advanced</i> shows each\
+ specific type of permission for finer control.'),
 dataset_acl_perms_type_options: [
                             {label:'Basic', value: 'BASIC'},
                             {label:'Advanced', value: 'ADVANCED'}
                           ],
-                          
+
 dataset_acl_user_placeholder: T('User'),
-dataset_acl_user_tooltip: T('Select the user to control the pool/dataset. Users\
- manually created or imported from a directory service\
- will appear in the drop-down menu.'),
-                          
+dataset_acl_user_tooltip: T('User account to which this ACL entry\
+ applies.'),
+
 dataset_acl_group_placeholder: T('Group'),
-dataset_acl_group_tooltip: T('Select the group to control the pool/dataset. Groups\
- manually created or imported from a directory service\
- will appear in the drop-down menu.'),
+dataset_acl_group_tooltip: T('Group to which this ACL entry applies.'),
 
 dataset_acl_uid_placeholder: T('User'),
-dataset_acl_uid_tooltip: T('Select the user to control the pool/dataset. Users\
- manually created or imported from a directory service\
- will appear in the drop-down menu.'),
+dataset_acl_uid_tooltip: T('User who controls the dataset. This user\
+ always has permissions to read or write the ACL and read or write\
+ attributes. Users created manually or imported from a directory service\
+ appear in the drop-down menu.'),
 
 dataset_acl_gid_placeholder: T('Group'),
-dataset_acl_gid_tooltip: T('Select the group to control the pool/dataset. Groups\
- manually created or imported from a directory service\
- will appear in the drop-down menu.'),
+dataset_acl_gid_tooltip: T('The group which controls the dataset. This\
+ group always has permissions to read or write the ACL and read or write\
+ attributes. Groups created manually or imported from a directory\
+ service appear in the drop-down menu.'),
```

Comment:
group@ does not always have permissions to read or write the ACL.
```
root@S4:/mnt/SHARE` # getfacl STUFF
# file: STUFF
# owner: root
# group: smbuser
            owner@:rwxp--aARWcCos:-------:allow
            group@:r-x---a-R-c--s:-------:allow
         everyone@:r-x---a-R-c--s:-------:allow
root@S4:/mnt/SHARE # su smbuser
$ setfacl -x 2 STUFF
setfacl: STUFF: acl_set_file() failed: Operation not permitted
```

Reply:
Aha, thanks for the clarification!